recipe/meta.yaml

```diff
@@ -1,54 +1,76 @@
-{% set version = "3.141.0" %}
+{% set version = "4.24.0" %}
 
 package:
   name: selenium
   version: {{ version }}
-
+#  Added 0000-no-rust-setup.patch to remove all references to Rust
+#  extensions from setup.py. This includes removing the import of setuptools_rust and
+#  the rust_extensions configuration, which resolves build failures caused by the unnecessary
+#   inclusion of Rust extensions.
 source:
+  patches:
+    - 0000-no-rust-setup.patch
   url: https://pypi.org/packages/source/s/selenium/selenium-{{ version }}.tar.gz
-  sha256: deaf32b60ad91a4611b98d8002757f29e6f2c2d5fcaf202e1c9ad06d6772300d
+  sha256: 88281e5b5b90fe231868905d5ea745b9ee5e30db280b33498cc73fb0fa06d571
 
 build:
-  number: 1000
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
+  skip: True  # [py<38]
   missing_dso_whitelist:  # [osx]
     - '*'                 # [osx]
 
 requirements:
   build:
     - {{ compiler('c') }}
+    - patch       # [not win]
+    - m2-patch    # [win]
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - urllib3
+    - urllib3 >=1.26,<3
```

Comment:
For other reviewers. It looks like `urllib3[socks]>=1.26,<3` will be expanded to:
```
    - urllib3 >=1.26,<3
    - pysocks >=1.5.6,<2.0,!=1.5.7
```
because our current `urllib3 2.2.2` has an optional dependency `pysocks` already available at runtime, see https://github.com/AnacondaRecipes/urllib3-feedstock/blob/master/recipe/meta.yaml#L30 and https://github.com/urllib3/urllib3/blob/2.2.2/pyproject.toml#L49-L51

Reply:
Thanks for clarifying! Just to confirm, no additional changes are needed on my end, correct?

Reply:
Correct